website/src/pages/cwkw/cwkw-layout.css.ts

```diff
@@ -70,7 +70,9 @@ export const openHeaderContents = style([
 export const banner = style([
   margin(0),
   paddingY(vspace.medium),
+
   {
+    width: `calc(${hsize.small} / 3)`,
```

Comment:
I'd use a fixed value added to the `hsize` object as a new field

Reply:
fixed!